kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -670,43 +674,43 @@ def compute_jumpdests(sections: list[KInner]) -> KInner:
     :return: This function iterates over each section, appending the jump destinations (0x5B) from the bytecode in a KAst Set.
     :rtype: KInner
     """
-    offset = 0
-    jumpdests = []
+    mutable_jumpdests = bytearray(b'')
     for s in sections:
         if type(s) is KApply and s.label == KLabel('buf'):
             width_token = s.args[0]
             assert type(width_token) is KToken
-            offset += int(width_token.token)
+            mutable_jumpdests += bytes(int(width_token.token))
         elif type(s) is KToken and s.sort == BYTES:
             bytecode = pretty_bytes(s)
-            jumpdests.extend(_process_jumpdests(bytecode, offset))
-            offset += len(bytecode)
+            mutable_jumpdests += _process_jumpdests(bytecode)
+            len(bytecode)
```

Comment:
Is it a typo?

Reply:
I mean line 686 and `len(bytecode)` specifically.

Reply:
yup, good catch! `len(bytecdeo)` was used to increment the `offset`. I removed the `offset` but I guess I did not catch this.